doc/algorithms_config/map_configs.ipynb

```diff
@@ -0,0 +1,502 @@
+    "|------------------|-------|--------|-----------------------------------------------------|\n",
+    "| retrain_full     | bool  | True   | if True, re-run the fit with the full training set, including data set aside for bump/sharpen validation.  If False, only use the subset defined via trainfrac fraction |\n",
+    "| trainfrac        | float | 0.75   | fraction of training data to use for training (rest used for bump thresh and sharpening determination)|\n",
+    "| seed             | int   | 1138   | Random number seed                                  |\n",
+    "| bumpmin          | float | 0.02   | minimum value in grid of thresholds checked to optimize removal of spurious small bumps |\n",
+    "| bumpmax          | float | 0.35   | max value in grid checked for removal of small bumps |\n",
+    "| nbump            | int   | 20     | number of grid points in bumpthresh grid search     |\n",
+    "| sharpmin         | float | 0.7    | min value in grid checked in optimal sharpening parameter fit |\n",
+    "| sharpmax         | float | 2.1    | max value in grid checked in optimal sharpening parameter fit |\n",
+    "| nsharp           | int   | 15     | number of search points in sharpening fit |\n",
+    "| max_basis        | int   | 35     | maximum number of basis funcitons to use in density estimate |\n",
+    "| basis_system     | str   | 'cosine' | type of basis sytem to use with flexcode |\n",
+    "| regression_params| dict  | {'max_depth': 8, 'objective': 'reg:squarederror'}| dictionary of options passed to flexcode, includes max_depth (int), and objective, which should be set  to reg:squarederror\n",
+    "| zmin                      | float | 0.0          | SHARED_PARAMS |\n",
+    "| zmax                      | float | 3.0          | SHARED_PARAMS |\n",
+    "| nzbins                    | int   | 301          | SHARED_PARAMS |\n",
+    "| nondetect_val             | float | 99.0         | SHARED_PARAMS |\n",
+    "| bands                     | array | [\"mag_u_lsst\", \"mag_g_lsst\", \"mag_r_lsst\", \"mag_i_lsst\", \"mag_z_lsst\", \"mag_y_lsst\"]      | SHARED_PARAMS |\n",
+    "| err_bands                 | array | [\"mag_err_u_lsst\", \"mag_err_g_lsst\", \"mag_err_r_lsst\", \"mag_err_i_lsst\", \"mag_err_z_lsst\", \"mag_err_y_lsst\"]| SHARED_PARAMS |\n",
+    "| mag_limits                | dict  | {mag_u_lsst:27.79, mag_g_lsst:29.04, mag_r_lsst:29.06, mag_i_lsst:28.62, mag_z_lsst:27.98, mag_y_lsst:27.05}| SHARED_PARAMS |\n",
+    "| ref_band                  | str   | \"mag_i_lsst\" | SHARED_PARAMS |\n",
+    "| redshift_col              | str   | \"redshift\"   | SHARED_PARAMS |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b647cb0b-0f2e-4b43-b98b-bb840edd637f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Estimate Params\n",
+    "\n",
+    "For flexzboost the inform parameters are more siginificant to obtain different results then the estimate parameters\n",
```

Comment:
I think it should be "than" instead of "then".

Reply:
fixed

Reply:
This change ("than" instead of "then") didn't appear here for me. Could you check if it was accomplished, please?

Reply:
yup, forgot a pull, tnx for remembering